CODEOWNERS

```diff
@@ -78,7 +78,7 @@ homeassistant/components/counter/* @fabaff
 homeassistant/components/cover/* @home-assistant/core
 homeassistant/components/cpuspeed/* @fabaff
 homeassistant/components/cups/* @fabaff
-homeassistant/components/daikin/* @fredrike @rofrantz
```

Comment:
Have you checked with @rofrantz about this?

Reply:
He mentioned a while back that he didn't have any interest in maintaining Daikin anymore.

Reply:
Ok. :+1: